src/component/card/dataverseCard/dataverseCard.tsx

```diff
@@ -1,22 +1,25 @@
 import type { FC } from 'react'
 import { useCallback } from 'react'
+import { useTranslation } from 'react-i18next'
+import { useNavigate } from 'react-router-dom'
 import './dataverseCard.scss'
 import { Card } from '@/component/card/card'
 import { Button } from '@/component/button/button'
 import { Tag } from '@/component/tag/tag'
 import ReactMarkdown from 'react-markdown'
 import type { ColorVariant } from '@/component/tag/tag'
-import { useTranslation } from 'react-i18next'
 
 type DataverseItem = 'dataspace' | 'dataset' | 'service'
 
 export type DataverseCardProps = {
+  id: string
   type: DataverseItem
   label: string
   description: string
 }
 
-export const DataverseCard: FC<DataverseCardProps> = ({ type, label, description }) => {
+export const DataverseCard: FC<DataverseCardProps> = ({ id, type, label, description }) => {
```

Comment:
If you are targeting a dataverse item, I'd call it: DataverseItemCard (and the props accordingly).

Reply:
you made a good point.